bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/styles.scss

```diff
@@ -6,6 +6,14 @@
   to { background-position: 0 0; }
 }
 
+.visuallyHidden {
+  position: absolute;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
+  height: 1px; width: 1px;
```

Comment:
What is the goal of styling a visually hidden element? Perhaps this is visible in some cases?

Reply:
The main reason for this is as a safeguard. It ensures that the visually hidden element won't enlarge to the point where it could be visible based on it's contents.